code/frameworks/nextjs/template/stories/Image.stories.jsx

```diff
@@ -78,8 +74,6 @@ export const Lazy = {
 export const Eager = {
   ...Lazy,
   parameters: {
-    // ignoring in Chromatic to avoid inconsistent snapshots since the image is sometimes not loaded in time
-    chromatic: { disableSnapshot: true },
```

Comment:
You can put this back if you want but I feel it's unlikely that these stories would cause inconsistent snapshots.

I've been snapshotting eager loaded images loaded with next/image using Chromatic for many years without images not being loaded in time.

What I think might've happened is what seemed like inconsistent snapshots was actually the bug we're trying to fix causing the snapshots to change.

Reply:
Looks like the lazy aspect can cause inconsistent snapshots, see the CI (chromatic)

Reply:
Since [the new snapshot](https://www.chromatic.com/test?appId=6398a69f2d9d21048ce8badc&id=650af5994d4736816a30d7a3) is correct (image isn't loaded), could you try rerunning the Chromatic build a few times and see if it changes? If it does, what we can do is either we can increase the height more than `200vh` until the image definitely doesn't get loaded, or we could just remove the Lazy story since lazy loading is the default state of `next/image` and thus doesn't really need to be tested.

Reply:
I've tried re-running it about 5 times, and it's consistently empty :(

Reply:
The `Lazy` snapshot should always be empty, while the `Eager` snapshot should always have the image showing. If that was the case after re-running 5 times then I think we can be are the snapshots are actually consistent!